gotham/src/extractor/mod.rs

```diff
@@ -1,8 +1,13 @@
 //! Extracts request data into type-safe structs using Serde.
 //!
-//! The request data is extracted by the `Route` implementation when dispatching the request. An
-//! application-provided data structure is used to deserialize the data and store it within the
-//! request `State` before the request is dispatched to the `Handler`.
+//! Extractors are added to route definitions when defining a `Router`. The `PathExtractor` and
+//! `QueryStringExtractor` traits provide usage examples. For further examples, see the Gotham
```

Comment:
Path and QueryString are actually standalone functionality groups now. Perhaps we just dump the "For further examples" sentence entirely.